lib/index.js

```diff
@@ -87,13 +87,13 @@ const setup = ({customTextFormattings, parseFigureProps, customCaption} = {}) =>
       return <Article items={[]} articleProps={articleProps} />;
     },
     shouldUpdate: ({props, id}, nextProps) => {
-      const current = currentElement[id];
-      const next = renderArticleJson({ items: nextProps.items });
-      nextElement[id] = next;
       const itemsHasUpdated = props.items !== nextProps.items;
-      const htmlHasUpdated = itemsHasUpdated && next.innerHTML !== current.innerHTML;
+      if (itemsHasUpdated) {
+        const next = renderArticleJson({ items: nextProps.items });
```

Comment:
but we do the rendering at https://github.com/micnews/article-json-to-contenteditable/pull/94/files#diff-6d186b954a58d5bb740f73d84fe39073R102 anyway right?